src/agents.jl

```diff
@@ -67,9 +67,9 @@ of the *Environment*
 abstract type Percept end
 
 """
-Given a *Percept* returns an *Action* apt for the agent.
+Given a *Percept* ? returns an *Action* apt for the agent.
```

Comment:
Not sure why the ? mark.

Reply:
Sentence was not meaningful, so I asked what would be there.

Reply:
So I asked you. Sorry if it is not the proper way, will keep in mind later on.